model_compression_toolkit/core/pytorch/back2framework/pytorch_model_builder.py

```diff
@@ -309,9 +309,19 @@ def _add_modules(self):
                 setattr(self, node.name, node_op)
             else:
                 self.add_module(node.name, node_op)
+            
+            if node.is_quantization_preserving() and self.get_activation_quantizer_holder is not None:
```

Comment:
instead of including `self.get_activation_quantizer_holder is not None` in each of the conditions you can just put it as an outer scope:
```
if self.get_activation_quantizer_holder is not None:
    if node.is_quantization_preserving()
    elif node.is_activation_quantization_enabled()
    ...
```

In addition, I see that this class implements a property `use_activation_holder_during_model_building` hich checks this exact thing, so I would recommand to use it instead of the `is None` statement